zppy/templates/slurm_header.sh

```diff
@@ -16,7 +16,7 @@
 # Running on cori-haswell or cori-knl depending on 'partition'
 
 #SBATCH  --job-name={{ prefix }}
-#SBATCH  --account=e3sm
+#SBATCH  --account={{ account }}
```

Comment:
@golaz Anvil's account is `condo`, not `e3sm`, so what should the default in `zppy/templates/default.ini` be?

Also, Chrysalis doesn't even include an account -- should it?

Reply:
Chrysalis defaults to `e3sm` so I have set it to use `e3sm` as a default.